### PR TITLE
add content to user manual

### PR DESCRIPTION
**Pull request by LimEunSang**
I made a simple instructions. If you give me any feedback, I'll work more.
I'd appreciate it if you could give me any suggestions.

### CODE REVIEW

File: user-manual/collection.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Collection
+
+## Favorite Albums
+
+You can add or delete albums to the list by clicking the star button.
+
+![Add Favorite Albums](https://i.imgur.com/xVp5iFO.png)
```
Comment (nukeop) [inline user-manual/collection.md]: Thanks for taking the screenshots, could you please make sure they're in their original size, so they're not blurry? And please crop them to the exact size of the window so there are no white borders.

**Comment by nukeop** [comment]
Thanks for this contribution, I left a comment with a possible improvement. I'll probably want to edit it a little to improve grammar etc, but I'll do that after merging.

File: user-manual/collection.md (hunk 1 above)
Comment (LimEunSang) [reply user-manual/collection.md]: I tried to modify it according to your feedback. The pixels in the current picture are 1385 x 918. If the picture is still blurry or broken, can you tell me the ideal pixel?

Comment (nukeop) [reply user-manual/collection.md]: Thanks, it looks great now. Would you like to add anything else, or is it ready to merge?

Comment (LimEunSang) [reply user-manual/collection.md]: I want to work more!

**Comment by LimEunSang** [comment]
I've completed the task. I'll wait for your feedback or merge.

**Pull request closed without merging**